l10n/en.js

```diff
@@ -63,14 +63,14 @@ OC.L10N.register(
     "You must enter at least a title for the new poll." : "You must enter at least a title for the new poll.",
     "Copy to clipboard: Ctrl+C, Enter" : "Copy to clipboard: Ctrl+C, Enter",
     "Thanks that you've voted. You can close the page now." : "Thanks that you've voted. You can close the page now.",
-    "Nothing selected!
-Click on cells to turn them green." : "Nothing selected!
-Click on cells to turn them green.",
-    "You are not registered.
-Please enter your name to vote
-(at least 3 characters!)" : "You are not registered.
-Please enter your name to vote
-(at least 3 characters!)",
+    "Nothing selected!" : "Nothing selected!",
+    "Click on cells to turn them green." : "Click on cells to turn them green.",
+    "Click on cells to turn them green." : "Click on cells to turn them green.",
+    "You are not registered." : " You are not registered",
+    "Please enter your name to vote" : "Please enter your name to vote",
+    "(at least 3 characters!)" : "You are not registered.",
+    "Please enter your name to vote" : "Please enter your name to vote",
+    "(at least 3 characters!)" : "(at least 3 characters!)",
```

Comment:
Will do, I was kind of confused why there were duplicates in the first place. But thanks for clarification.